chemie/elections/views.py

```diff
@@ -375,3 +390,64 @@ def admin_end_election(request):
             )
     election.end_election()
     return redirect('elections:results')
+
+
+@permission_required('elections.add_election')
+@login_required
+def change_rfid_status(request):
+    if request.method == "POST":
+        form = GetRFIDForm(request.POST)
+        if form.is_valid():
+            rfid = form.cleaned_data.get('rfid')
+            em_code = ProfileManager.rfid_to_em(rfid)
+            try:
+                profile = Profile.objects.get(access_card=em_code)
+            except:
+                messages.add_message(request, messages.WARNING, 'Studentkortnummeret er ikke registrert enda.')
+                return redirect('elections:add_rfid')
+            eligible = profile.eligible_for_voting
+            if eligible:
+                profile.eligible_for_voting = False
+            else:
+                profile.eligible_for_voting = True
+            profile.save()
+            status = profile.eligible_for_voting
+            if status:
+                messages.add_message(request, messages.SUCCESS, '{} har sjekket inn'.format(profile.user.get_full_name()))
+            else:
+                messages.add_message(request, messages.WARNING, '{} har sjekket ut'.format(profile.user.get_full_name()))
+        return redirect('elections:checkin')
+    else:
+        is_open = election_is_open()
+        form = GetRFIDForm()
+        context = {'form': form, 'is_open': is_open}
+    return render(request, 'elections/check_in.html', context)
+
+
+@permission_required('elections.add_election')
```

Comment:
Kanskje flytte heile viewet inn i `customprofile`? Da kan alle andre apps som kreve kortnr. dra nytte av samme funksjonalitet, feks bedpres / events. 

Da må man trolig lage en custom permission (ala `can_edit_access_card` eller nokke) i stede for `add_election`.

Reply:
Enig =)

Reply:
5d68b213402bebe91adab691791461ec01868751